src/pages/Subscription/Subscription.jsx

```diff
@@ -11,6 +11,14 @@ export default function SubscriptionPage() {
     window.alert(inputState);
   }
 
+  function handleRevertButton() {
+    window.alert(revertButton(inputState));
+  }
+
+  function revertButton(s) {

```

Comment:
Never abbreviate the variable, you are limiting the meaning, the ideal is always to use a full name.
Because In that case, `s` doesn't mean anything. 

```suggestion
  function revertButton(text) {
```